packages/affirm-integration/project.json

```diff
@@ -0,0 +1,23 @@
+{
+  "root": "packages/affirm-integration",
+  "sourceRoot": "packages/affirm-integration/src",
+  "projectType": "library",
+  "targets": {
+    "lint": {
+      "executor": "@nrwl/linter:eslint",
+      "outputs": ["{options.outputFile}"],
+      "options": {
+        "lintFilePatterns": ["packages/affirm-integration/**/*.ts"]
+      }
+    },
+    "test": {
+      "executor": "@nrwl/jest:jest",
+      "outputs": ["coverage/packages/affirm-integration"],
+      "options": {
+        "jestConfig": "packages/affirm-integration/jest.config.js",
+        "passWithNoTests": true
+      }
+    }
+  },
+  "tags": []
```

Comment:
```suggestion
  "tags": [
    "scope:integration",
    "scope:shared"
    ]
```

Reply:
@bc-peng according to discussion in` #payment-integration-frontend` slack channel https://bigcommerce.slack.com/archives/C02TV6E3BM5/p1709063130646769 I will leave `scope:integration` only ok?